skfda/exploratory/visualization/_outliergram.py

```diff
@@ -139,18 +156,86 @@ def modified_epigraph_index_list(self) -> np.ndarray:
             - self.fdata.domain_range[0][0]
         )
 
-        function = rankdata(
+        # Array containing at each point the number of curves
+        # are above it.
+        num_functions_above = rankdata(
             -self.fdata.data_matrix,
             method='max',
             axis=0,
         ) - 1
 
         integrand = integrate.simps(
-            function,
+            num_functions_above,
             x=self.fdata.grid_points[0],
             axis=1,
         )
 
         integrand /= (interval_len * self.fdata.n_samples)
 
         return integrand.flatten()
+
+    def __compute_distances(self) -> np.ndarray:
```

Comment:
I prefer protected methods (one underscore) rather than private ones (two underscores) unless there is a very good reason. Private names have a little of "magic" by prepending the class name.